plugin/trino-cassandra/src/test/java/io/trino/plugin/cassandra/TestCassandraConnectorTest.java

```diff
@@ -242,97 +237,305 @@ public void testCharVarcharComparison()
     @Test
     public void testPushdownUuidPartitionKeyPredicate()
     {
-        assertThat(query(format("SELECT col_text FROM %s.%s WHERE col_uuid = UUID '00000000-0000-0000-0000-000000000001'", KEYSPACE, TABLE_PUSHDOWN_UUID_PARTITION_KEY_PREDICATE)))
```

Comment:
Please remove `TABLE_PUSHDOWN_UUID_PARTITION_KEY_PREDICATE` and other unused tables from `CassandraTestingUtils`.

Reply:
Good catch. I hadn't even noticed it before.